### PR TITLE
docs - connecting to dbs with ssl certificates

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
@paoliniluis, here's the write-up from our meeting on connecting to databases with SSL certificates. I also included the use case for setting up the connection with environment variables.

Let me know if I'm missing anything here, and thanks again for walking me through it.

### CODE REVIEW

File: docs/administration-guide/secure-database-connections-with-ssl-certificates.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+## Securing database connections using an SSL certificate
+
+If you'd like to connect your Metabase Cloud or self-hosted instance to a database, you can secure the connection using Secure Socket Layer (SSL) encryption with a certificate.
+
+Why you'd want to do this:
+
+- You're using Metabase Cloud and want to create a secure connection to a data warehouse that accepts a JDBC connection (e.g., PostgreSQL).
```
Comment (paoliniluis) [inline docs/administration-guide/secure-database-connections-with-ssl-certificates.md:7]: I would say here: You're using Metabase Cloud and want to ensure the identity of the Data Warehouse you're connecting to (e.g., PostgreSQL, MySQL).

File: docs/administration-guide/secure-database-connections-with-ssl-certificates.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- You're self-hosting Metabase and want to create a secure connection to an application database hosted by an external provider (this situation is rare).
```
Comment (paoliniluis) [inline docs/administration-guide/secure-database-connections-with-ssl-certificates.md:8]: I would say here: You're self-hosting Metabase and want to ensure the identity of Data Warehouse hosted by an external provider.

File: docs/administration-guide/secure-database-connections-with-ssl-certificates.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+If you're using Metabase Cloud, the application database is handled for you, so you'd only need to secure connections to data warehouses that you add to your Metabase.
+
+### Prerequisites
+
+A database that allows a JDBC connection, as you'll need to use a connection string to secure the connection.
```
Comment (paoliniluis) [inline docs/administration-guide/secure-database-connections-with-ssl-certificates.md:14]: you'll need to use a connection string to specify the certificate that you want to use

File: docs/administration-guide/secure-database-connections-with-ssl-certificates.md (hunk 2 above)
Comment (paoliniluis) [reply docs/administration-guide/secure-database-connections-with-ssl-certificates.md:8]: or you can also use this to ensure you're using the most strict connection parameters when connecting to your application database

File: docs/administration-guide/secure-database-connections-with-ssl-certificates.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+If you're using Metabase Cloud, the application database is handled for you, so you'd only need to secure connections to data warehouses that you add to your Metabase.
+
+### Prerequisites
+
+A database that allows a JDBC connection, as you'll need to use a connection string to specify the certificate you want to use.
+
+### Step 1: Download the root certificate from your provider
+
+If you're running Metabase via a Docker container, you should already have the certificates for AWS and Azure. Otherwise, go to your external provider and find a link to download the root certificate for connecting to your database.
```
Comment (paoliniluis) [inline docs/administration-guide/secure-database-connections-with-ssl-certificates.md:18]: We can include here that the Azure certificate is under "/app/certs/DigiCertGlobalRootG2.crt.pem" and the AWS RDS one is in "/app/certs/rds-combined-ca-bundle.pem" in Metabase's docker image. If users need any other, they can build their own Docker image

**Comment by paoliniluis** [comment]
just added 1 more comment, but looks good to me!

**Review by dragonsahead — APPROVED** [review]

**Pull request merged**